site/content/installation/running-on-openshift.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Deploying NGINX Gateway Fabric in an Openshift environment"
+# Change draft status to false to publish doc
+draft: false
+description: "Learn how to run NGINX Gateway Fabric in an Openshift environment."
+weight: 400
+toc: true
+tags: [ "docs" ]
+docs: "DOCS-0000"
+---
+
+{{< custom-styles >}}
+
+## Prerequisites
+
+To complete the steps in this guide, you first need to install the following tools for Kubernetes management and development:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/): A command-line interface for Kubernetes that allows you to manage and inspect cluster resources, and control containerized applications.
+- Access to an Openshift environment with cluster administrative permissions.
+
+
+## Create SCC Object
```

Comment:
This doc will need to be updated. We should generate the SCC to add to our manifests for users that install using those instead of helm. That way we can support both installation methods

Reply:
Hmm, yea everything is changing so much with the templating, I realized with the Helm installation, the installation would be the same except adding the "--set platform=openshift", so I was considering removing the dedicated Openshift document. However I was waiting to see what the manifest generation would look like and the installation process too, so once I get that done I'll revisit the docs.

Reply:
I've been able to generate the SCC to add to our manifests, but I'm struggling on the best way to add the rbac part. 

this little chunk in the clusterrole  from the helm template:

```
{{- if eq .Values.platform "openshift" }}
- apiGroups:
  - security.openshift.io
  resources:
  - securitycontextconstraints
  resourceNames:
  - {{ include "nginx-gateway.scc-name" . }}
  verbs:
  - use
{{- end }}
```

I don't see many options outside of:
1. Add a nginx-gateway-openshift.yaml + plus version + experimental version with that part included
2. Just default that option to be on, so all the manifests will contain that rbac in the clusterrole


any thoughts?

Reply:
Ugh, manifests are annoying to support lol. Maybe just an `deploy/manifests/openshift` directory that contains all the same stuff. Curious what others think

Reply:
So that would be option 1, but just in a different directory?

Reply:
Yes